cartconditions/Coupon.php

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
This is no longer needed right?

Reply:
Yep its not needed - sorry I havent updated this PR as I was waiting to see how we progressed with the flame side of the calculation